core/types/confirmation/state.go

```diff
@@ -13,6 +13,9 @@ const (
 	// Accepted is the state for accepted entities.
 	Accepted
 
+	// NotConflicting is the state for a conflict, whose all conflicting conflicts are orphaned and rejected.
+	NotConflicting
```

Comment:
Can we move this whole thing to GoShimmer? This is too specific to be part of hive

Reply:
Done